tools/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suppressions PUBLIC "-//Puppy Crawl//DTD Suppressions 1.1//EN" "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+<suppressions>
+    <suppress files="build.properties" checks="AboutHtmlCheck" />
+    <suppress files=".classpath" checks="MavenPomderivedInClasspathCheck" />
+    <!-- These suppressions define which files to be suppressed for which checks. -->
+    <suppress files=".+[\\/]internal[\\/].+\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck|NullAnnotationsCheck"/>
+    <suppress files=".+DTO\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck" />
+    <suppress files=".+Impl\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck"/>
```

Comment:
".Impl" is not allowed as file extension for extensions in this repo afaik

Reply:
It's `fooBarImpl.java` which sounds valid to me.

Reply:
yup true. So the NullAnnotationsChecks are disabled for Impl. But why? I think that line at least needs a comment.